vault/resource_jwt_auth_backend.go

```diff
@@ -106,6 +123,35 @@ func jwtAuthBackendResource() *schema.Resource {
 	}
 }
 
+func jwtCustomizeDiff(d *schema.ResourceDiff, meta interface{}) error {
+	var _ interface{}
+	var discUrlExists, jwksUrlExists, jwtPubKeysExists bool
+	_, discUrlExists = d.GetOk("oidc_discovery_url")
+	_, jwksUrlExists = d.GetOk("jwks_url")
+	_, jwtPubKeysExists = d.GetOk("jwt_validation_pubkeys")
+
+	if (discUrlExists && (jwksUrlExists || jwtPubKeysExists)) || (jwksUrlExists && (discUrlExists || jwtPubKeysExists)) || (jwtPubKeysExists && (discUrlExists || jwksUrlExists)) {
```

Comment:
Isn't this check handled by `ConflictsWith` on the fields?

Reply:
You're right, as written this check is redundant.  What I had intended to do was replace the check that was originally part of the update method to ensure that at least one of these properties is set.